imu_filter_madgwick/CMakeLists.txt

```diff
@@ -64,4 +68,9 @@ endif()
 ament_export_include_directories(include)
 ament_export_libraries(imu_filter_madgwick)
 
+install(
+  DIRECTORY include/${PROJECT_NAME}/
+  DESTINATION include/${PROJECT_NAME}
+)
+
```

Comment:
Normally, it would not be necessary, when I compiled it on both my machines, the include directory did not went to the install directory and the find_package(imu_filter_madgwick) failed with an error that the include directory was missing. Let's think of theses line as a failsafe if the `ament_export_include_directories(include)` did not work, which seems to not be the case. 

If you're talking about, I think that the faulty one since it create this:
```cmake
install(
  DIRECTORY include/
  DESTINATION include
)
```
`~/ros2/install/imu_filter_madgwick/include/include`
while the other creates
`~/ros2/install/imu_filter_madgwick/include/imu_filter_madgwick`

I'm removing the one without the ${PROJECT NAME}

Reply:
You were missing the '/' after `include`. This version works:

```cmake
install(
  DIRECTORY include/
  DESTINATION include
)
```

This doesn't:

```cmake
install(
  DIRECTORY include
  DESTINATION include
)
```

I've committed the first version in d8128147.